### PR TITLE
fix(header): bump that z-index a little

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: src/pivotal-ui/components/header/header.scss [hunk 1]
```diff
@@ -52,7 +52,7 @@ View the [regular header on its own page](/header.html) to see mobile sizing fea
 */
 
 header {
-  z-index: 0; // A separate rendering context because browsers are 💩 and full of bugs.
+  z-index: 2; // A separate rendering context because browsers are 💩 and full of bugs, and 2 because 1 is the dropdown menu overlay and everything is terrible
```
Comment (stubbornella) [inline src/pivotal-ui/components/header/header.scss]: we might have fewer bugs if we use the built in z-index from bootstrap.

**Comment by stubbornella** [comment]
This is how z-index is meant to be modified: https://github.com/npm/pivotal-ui/blob/master/src/pivotal-ui/components/pui-variables.scss#L642

File: src/pivotal-ui/components/header/header.scss (hunk 1 above)
Comment (stubbornella) [reply src/pivotal-ui/components/header/header.scss]: https://github.com/npm/pivotal-ui/blob/master/src/pivotal-ui/components/pui-variables.scss#L642

**Comment by aredridel** [comment]
Using the variable, though there's no documentation as to why you'd choose which one of those, so it seems pretty arbitrary still.

**Pull request merged**